python/ray/autoscaler/aws/example-subnets.yaml

```diff
@@ -0,0 +1,30 @@
+cluster_name: subnets
+
+max_workers: 1
+
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+
+auth:
+    ssh_user: ubuntu
+
+# If required, head and worker nodes can exist on different subnets and
+# communicate via VPC peering.
```

Comment:
Hmm, I don't think this is correct. For two subnets within the same VPC, both subnets just need a route. I think VPC peering is only necessary if you want two different VPCs to be able to talk in between each other, yeah?

Did you mean the later?

Reply:
Right - good catch. This should instead read:

```
# If required, head and worker nodes can exist on different VPCs and
# communicate via VPC peering.
```

Reply:
Or perhaps, more specifically:
```
# If required, head and worker nodes can exist on subnets in different VPCs and
# communicate via VPC peering.
```